package.json

```diff
@@ -22,7 +22,7 @@
   "scripts": {
     "build": "tsc && tsc -p tsconfig.esm.json",
     "lint": "prettier --check \"src/**/*.ts\"",
-    "prepublishOnly": "npm run build",
+    "prepack": "npm run build",
```

Comment:
Is this change required?

Reply:
This was only required in order to be able to `yarn install` directly from this fork. The `newLine` setting was in order to avoid inconsistent output and checksum mismatches on Windows.

However, after making this PR I realized there was an older one which addresses the same `file` issue. Please take a look: #13